specification/communication/resource-manager/Microsoft.Communication/preview/2023-03-01-preview/SenderUsernames.json

```diff
@@ -0,0 +1,333 @@
+        ],
+        "x-ms-examples": {
+          "Create or update SenderUsernames resource": {
+            "$ref": "./examples/senderUsernames/createOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Operation to delete a SenderUsernames resource.",
+        "operationId": "SenderUsernames_Delete",
+        "summary": "Delete",
+        "tags": [
+          "SenderUsernames"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "EmailServices.json#/parameters/EmailServiceName"
+          },
+          {
+            "$ref": "Domains.json#/parameters/DomainName"
+          },
+          {
+            "$ref": "#/parameters/SenderUsername"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The delete operation was successful."
+          },
+          "404": {
+            "description": "The resource did not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete SenderUsernames resource": {
+            "$ref": "./examples/senderUsernames/delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SenderUsernameResource": {
+      "description": "A class representing a SenderUsername resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SenderUsernameProperties",
+          "description": "The properties of a SenderUsername resource.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SenderUsernameResourceCollection": {
+      "description": "A class representing a Domains SenderUsernames collection.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of SenderUsernames",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SenderUsernameResource"
+          }
+        },
+        "nextLink": {
+          "description": "The URL the client should use to fetch the next page (per server side paging).\r\nIt's null for now, added for future use.",
```

Comment:
>r\nIt's null for now, added for future use.", [](http://example.com/codeflow?start=105&length=45)

I'm not sure of the value of calling this out, if you want people to implement things in future proof ways.